settings/defaults.py

```diff
@@ -1,10 +1,22 @@
+# New tokens can be found at https://archive.org/account/s3.php
+
+DOI_FORMAT = '10.70102/fk2osf.io/{guid}'
+
+DATACITE_USERNAME = ''
+DATACITE_PASSWORD = ''
+DATACITE_URL = 'https://doi.test.datacite.org/'
+DATACITE_PREFIX = '10.70102'  # Datacite's test DOI prefix -- update in production
+
 CHUNK_SIZE = 1000
-OSF_API_URI = 'http://localhost:8000/'
+PAGE_SIZE = 100
+OSF_API_URI = 'https://localhost:8000/'
 OSF_COLLECTION_NAME = 'cos-dev-sandbox'
+OSF_BEARER_TOKEN = ''
 
-IA_ACCESS_KEY = 'change to valid token'
-IA_SECRET_KEY = 'change to valid token'
+IA_ACCESS_KEY = ''
+IA_SECRET_KEY = ''
 
 OSF_API_URL = 'https://localhost:8000/'
+OSF_FILES_URL = 'https://localhost:8000/'
```

Comment:
This is the same as `OSF_API_URL` and incorrect, the actual "files" url is waterbutler at `https://localhost:7777/`